packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -119,10 +119,14 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
   onRenderLowerContent?: IRenderFunction<IComboBoxProps>;
 
   /**
-  * Custom width for dropdown. If value is 0, width of the input field is used.
-  * @default 0
+  * Custom width for dropdown. If value is 0, width of the input field is used (unless useComboBoxAsMenuWidth is undefined or false)

```

Comment:
Is the comment still true? I don't think the value of zero has a special meaning anymore. We might also want to warn if  dropDownWidth and useComboBoxAsMenuWidth are both set

Reply:
It's also debatable that this prop should go away. The width of the menu is perhaps a styling concern and not a component prop, we could (and should) leverage merge styles for customizing the menu width. Matching the width of the input box is a different story, that requires javascript so that could probably remain a prop. Probably out of scope for this PR anyway.

Reply:
Yeah, that's what I was thinking especially since you could already define the desired width via the CalloutProps that are available at the comboBox level, but I agree that's out of scope for this PR